docs/platforms/python/configuration/options.mdx

```diff
@@ -57,6 +57,12 @@ The `environment` config option takes precedence over the environment variable.
 
 </ConfigKey>
 
+<ConfigKey name="dist">
+
+The distribution of the application. Distributions are used to disambiguate build or deployment variants of the same release of an application. The dist can be for example a build number.
```

Comment:
```suggestion
Application distribution. This can be used to disambiguate build or deployment variants that are part of the same release. (For example, the dist can be a build number.)
```